contracts/src/Rollup.sol

```diff
@@ -64,6 +64,34 @@ abstract contract RollupBase is
         __Ownable_init();
         __UUPSUpgradeable_init();
     }
+
+    /// @inheritdoc IRollup
+    function setConfirmationPeriod(uint256 newPeriod) public override onlyOwner {
+        confirmationPeriod = newPeriod;
```

Comment:
Please introduce sanity checks here.
The `newPeriod` should be in a range between `minConfirmationPeriod` and `maxConfirmationPeriod`.

If nothing, make sure that `newPeriod` != 0.

However, I think we will first need to discuss what kind of checks do we need to implement if at all we need to implement them.

cc: @ujvl @LEAFERx